docs/install/conda.rst

```diff
@@ -14,9 +14,24 @@ option when installing conda.
 --------------------------
 Binary Package (For Users)
 --------------------------
+
+To list all of the versions of PyNE available on `conda-forge
+<https://conda-forge.github.io/>`_ channel, in your terminal window run::
+
+    conda search -c conda-forge pyne
+
 Binary distributions of the latest release for mac and linux (64-bit) 
 using the conda package manager can be installed by running the command::
 
     conda install -c conda-forge pyne
 
+If you want to install PyNE with the correct package specification, try
+``pkg_name=version=build_string``.
+
+For example, if you want to install ``pyne version=0.7.0`` with build option ``moab_openmc_py36he21c9c4_1``, you would enter::
```

Comment:
It may be more interesting to users to use wild card on specifying the build options.  That is, a user may know that they want a version with moab and openmc, but don't care about the hash or python version.  Therefore, they might install as:
` conda install -c conda-forge pyne=0.7.0=moab_openmc*`